.github/workflows/nrp-combo-test.yml

```diff
@@ -1,4 +1,4 @@
-name: Universal NRP Test
+name: Universal NRP and Modules Combo Test
```

Comment:
(Spoke offline) The name change will break our internal build pipelines

Reply:
(Spoke offline) We need to follow-up with a PR for the internal build pipelines to update the name of this workflow.